app/coffee/PersistenceManager.coffee

```diff
@@ -28,6 +28,7 @@ module.exports = PersistenceManager =
 				sendImmediately: true
 			jar: false
 			timeout: MAX_HTTP_REQUEST_LENGTH
+			maxAttempts: 2  # for requestretry
```

Comment:
Good call on the timeout.

The issue was network timeouts, which I think are covered by the default strategy.